src/olympia/amo/tests/ui/setup.cfg

```diff
@@ -0,0 +1,4 @@
+[pytest]
+addopts = -r a --verbose --junit-xml=results.xml --html=results.html
+base_url = https://addons-dev.allizom.org
+sensitive_url = mozilla\.(com|org)
```

Comment:
So the reason this is needed is because if there's just one setup.cfg then pytest makes that the root directory, and discovers tests from there. This means the root conftest.py is found and this causes an import error because django and other packages are not installed (and should not be required for running the UI tests). I've discussed this with @EnTeQuAk on IRC and I'm going to look into moving the UI tests to a top level directory so they're clearly separated from the existing tests.